apps/browser/src/sys/logger/BrowserLogger.ts

```diff
@@ -5,28 +5,25 @@ export interface BrowserLoggerConfig {
 }
 
 export class BrowserLogger implements ILogger {
-  constructor(
-    private _config: BrowserLoggerConfig
-  ) { }
+  constructor(private _config: BrowserLoggerConfig) {}
 
-  info(info: string): void {
-    this._config.onLog(info)
+  async info(info: string): Promise<void> {
```

Comment:
Yes, because we're using the `Workspace` in evo for logging, which is async
E.g. on the backend, all logging will be done either to the DB or a Supabase bucket (or a third party service)

Reply:
Okay nice